attestation-service/attestation-service/src/bin/restful-as.rs

```diff
@@ -49,17 +50,30 @@ enum WebApi {
     Policy,
 }
 
+#[derive(Error, Debug)]
+pub enum RestfulError {
+    #[error("Creating service failed")]
+    Service(#[from] ServiceError),
+    #[error("Failed to read AS config file")]
+    Config(#[from] ConfigError),
+    #[error("Openssl errorstack")]
```

Comment:
we'd better add some more details about the error, or include the original error inside the error information, s.t.
```rust
#[error("HTTPS set failed due to openssl error: {0}")]
```